tools/clang/lib/SPIRV/EmitVisitor.cpp

```diff
@@ -1437,10 +1437,11 @@ void EmitVisitor::generateDebugSourceContinued(uint32_t textId,
 void EmitVisitor::generateChoppedSource(uint32_t fileId, SpirvDebugSource *inst) {
   // Chop up the source into multiple segments if it is too long.
   llvm::SmallVector<std::string, 2> choppedSrcCode;
-  std::string text;
+  std::string text = inst->getContent();
```

Comment:
text seem to only be used in the if(spcOptions.debugInfoSource) scope.
Would suggest moving declaration into the scope.